pnpbridge/docs/schemas/RootPnpBridgeInterfaceModel.json

```diff
@@ -0,0 +1,39 @@
+{

```

Comment:
Where is this interface definition used?

Reply:
It is the "RootInterfaceModelId" which is a part of the struct CONNECTION_PARAMETERS, used to create the iot device client handle in IotComms_CreateIotDeviceHandle.

---
In reply to: [461975907](https://github.com/Azure/AzurePnPBridgePreview/pull/34#discussion_r461975907) [](ancestors = 461975907)